docker-compose.yml

```diff
@@ -32,11 +32,11 @@ services:
     networks:
       - strapi-app-network
     volumes:
-      - ./app:/srv/app
+      - ./strapi-app:/srv/strapi-app
```

Comment:
Do you know if the strapi config relies on the name app?  i.e. does this still work? 
The alternative would be ` ./strapi-app:/srv/app` instead of ` ./strapi-app:/srv/strapi-app`

Reply:
no clue at all hmm
ill look into it!